mautic/mautic
# Show labels of select, multiselect and boolean custom lead fields

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
We noticed lead profile display values of select, multiselect or boolean fields. But custom fields have labels to values and these labels are more human friendly. This PR normalize these values and allow display in friendly format. 


Now

![image](https://user-images.githubusercontent.com/462477/69749983-f347a680-114b-11ea-9a93-89779f0f459e.png)

Expected

![image](https://user-images.githubusercontent.com/462477/69750140-56393d80-114c-11ea-9ec8-754d7533b3bb.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create custom field select, multiselect and boolean (all with different labels / values)
2. Create contact and set these fields
3. Go to contact profile and see values there

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat all steps
3. See if works properly
